src/main/java/io/vertx/core/VertxOptions.java

```diff
@@ -675,6 +675,20 @@ public VertxOptions setDisableTCCL(boolean disableTCCL) {
     return this;
   }
 
+  /**
+   * Return the {@link ThreadOptions} used to create {@link io.vertx.core.impl.VertxThread}.
+   */
+  public ThreadOptions getThreadOptions() {
```

Comment:
I think only the `daemon` setting is necessary, priority can always be set by the thread factory and will not be overwritten

Reply:
So you mean we should only provide a daemon setting here, without touching the VertxThreadFactory, but instead changing the code in VertxImpl to reset the daemon field of the VertxThread?